### PR TITLE
Annotation tooltip apply all and category

### PR DESCRIPTION
**Pull request by fondation451**
Issue: https://github.com/fondation451/label/issues/190

## Changes
- Change for category
- Apply for all
- The category are now sorted in the left panel

### CODE REVIEW

File: packages/generic/client/src/pages/Home/DocumentAnnotator/AnnotationsPanel/lib/groupAnnotations.ts [hunk 1]
```diff
@@ -4,20 +4,27 @@ import { fetchedAnnotationType } from '../../../../../types';
 export { groupAnnotations };
 
 function groupAnnotations(annotations: fetchedAnnotationType[]) {
-  const annotationsByCategory: { [category: string]: { annotation: fetchedAnnotationType; occurences: number }[] } = {};
+  console.log('groupAnnotations');
```
Comment (BenoitSerrano) [inline packages/generic/client/src/pages/Home/DocumentAnnotator/AnnotationsPanel/lib/groupAnnotations.ts:7]: console.log furtif

File: packages/generic/client/src/pages/Home/DocumentAnnotator/DocumentPanel/AnnotationTooltipMenu.tsx [hunk 2]
```diff
@@ -15,6 +15,7 @@ function AnnotationTooltipMenu(props: {
   onClose: () => void;
   open: boolean;
 }): ReactElement {
+  const [shouldApplyEveryWhere, setShouldApplyEveryWhere] = useState(false);
```
Comment (BenoitSerrano) [inline packages/generic/client/src/pages/Home/DocumentAnnotator/DocumentPanel/AnnotationTooltipMenu.tsx:18]: everywhere en un seul mot, pas de camelCase au milieu

File: packages/generic/client/src/pages/Home/DocumentAnnotator/AnnotationsPanel/lib/groupAnnotations.ts (hunk 1 above)
Comment (fondation451) [reply packages/generic/client/src/pages/Home/DocumentAnnotator/AnnotationsPanel/lib/groupAnnotations.ts]: @BenoitSerrano 
Done 13a5c8e

File: packages/generic/client/src/pages/Home/DocumentAnnotator/DocumentPanel/AnnotationTooltipMenu.tsx (hunk 2 above)
Comment (fondation451) [reply packages/generic/client/src/pages/Home/DocumentAnnotator/DocumentPanel/AnnotationTooltipMenu.tsx]: @BenoitSerrano 
Done 8d01316

**Pull request merged**